e2e/playwright/tests/change-channel/test.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { test, expect } from '@playwright/test';
+import { login, uploadLicense } from '../shared';
+
+const CUSTOMER_ID = '2k6jemHbYgZFqtwgyjqiVfjRQqi';
+const APP_ID = '2k6j65t0STtrZ1emyP5PUqBIQ23';
+const AUTOMATED_CHANNEL_ID = '2k6j62KPRQLjO0tF9zZB6zJJukg';
+const ALTERNATE_CHANNEL_ID = '2k6j61j49IPyDyQlbmRZJsxy3TP';
+
+test('change channel', async ({ page }) => {
+    test.slow();
+    await changeChannel(AUTOMATED_CHANNEL_ID);
+    await login(page);
+    await uploadLicense(page, expect);
+    await page.getByRole('button', { name: 'Deploy' }).click();
+    await expect(page.locator('#app')).toContainText('Automated');
+    await changeChannel(ALTERNATE_CHANNEL_ID);
+
+    await page.getByText('Sync license').click();
+
+    await expect(page.getByLabel('Next step')).toContainText('License synced', { timeout: 10000 });
+    await page.getByRole('button', { name: 'Ok, got it!' }).click();
+
+    await expect(page.locator('#app')).toContainText('Alternate');
+    await expect(page.locator('#app')).toContainText('1.0.3', { timeout: 10000 });
+    await expect(page.locator('#app')).toContainText('Upstream Update', { timeout: 10000 });
+
+    await page.waitForTimeout(5000);
```

Comment:
why is this needed?

Reply:
as sorry , this may actually not be needed anymore since just removed the preflight checks from the alternate channel. Originally was using this to try and give the preflight checks a chance to complete. Let me remove it and see if theres remaining timing issues.